docs/spark/spark-ddl.md

```diff
@@ -171,6 +171,14 @@ Iceberg uses table properties to control table behavior. For a list of available
 ALTER TABLE prod.db.sample UNSET TBLPROPERTIES ('read.split.target-size')
 ```
 
+To set table comment using `SET TBLPROPERTIES`:
```

Comment:
I think this should be "`SET TBLPROPERTIES` can also be used to set the table comment (description):"

Reply:
+1 to Ryan's suggestion. That reads better in my opinion.

Reply:
thanks @kbendick @rdblue , fixed.

Reply:
Do we need `(description)` there? I think table comment is straightforward.

Reply:
I also think that (description) is a bit unnecessary given that comment is already there.